docs/usage/faq.md

```diff
@@ -39,6 +39,24 @@ The Renovate team will only create bugfixes for an older version if the hosted a
 If you're using the hosted app, you don't need to do anything, as the Renovate maintainers update the hosted app regularly.
 If you're self hosting Renovate, use the latest release if possible.
 
+## Renovate core features not supported on all platforms
+
+| Feature                                 | Platforms which lack feature                      | See Renovate issue(s) |
+| --------------------------------------- | ------------------------------------------------- | --------------------- |
+| Dependency Dashboard                    | BitBucket, BitBucket Server, Azure                | #9592                 |
+| Hosted app                              | GitLab, BitBucket, BitBucket Server, Azure, Gitea |                       |
+| Cloning repo over SSH selfhosted        | GitLab                                            | #5406                 |
+| Configurable merge strategy and message | BitBucket, BitBucket Server                       | #7184                 |
```

Comment:
I think this is also unsupported for all platforms, but will be implemented for bitbucket first